src/Symfony/Bridge/Twig/Mime/TemplatedEmail.php

```diff
@@ -67,6 +67,18 @@ public function getContext(): array
         return $this->context;
     }
 
+    public function isRendered(): bool
+    {
+        return null === $this->htmlTemplate && null === $this->textTemplate;
```

Comment:
Why not using a boolean property as done in the other class?
Setting those to `null` in the next method could lead to inconsistency?